protocol/prep_encoder.go

```diff
@@ -41,7 +41,8 @@ func (pe *prepEncoder) putInt64(in int64) {
 }
 
 func (pe *prepEncoder) putUUID(in uuid.UUID) error {
-	return errors.New("not implemented")
+	pe.length += 16
+	return nil
```

Comment:
@vladoatanasov  is this related to the well kept secret header size that we talked earlier sir?

Reply:
no, this function encodes a UUID type into the Kafka protocol, which encodes to a 16 byte code. https://kafka.apache.org/protocol.html#protocol_types